docs/BTCPayVsOthers.md

```diff
@@ -113,8 +113,6 @@ As the BTCPay Server community continues to grow, more deployment methods, use c
 
 Currently, BTCPay Server is a processor **without fiat conversion** capabilities out of the box. As a merchant, this may be difficult if business costs require fiat. Not providing fiat conversion out of the box allows BTCPay Server merchants to avoid KYC and AML identification verification. This also allows BTCPay Server to be free and available for anyone to use.
 
-However, a fiat conversion plugin is available for BTCPay Server. Since merchants are always the owners of their private keys, they can always freely convert their coins manually, but for now there's no instant-fiat conversion. More info on the [Transmuter](https://github.com/btcpayserver/btcTransmuter).
```

Comment:
Nit: we have some plugins now that do offer this functionality. We don't need to explicitly list them but mention they should explore plugins